clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/KafkaStoreIngestionService.java

```diff
@@ -1195,7 +1195,8 @@ public void updatePartitionOffsetRecords(String topicName, int partition, ByteBu
       LOGGER.error(
           "Caught exception when deserializing offset record byte array: {} for replica: {}",
           Arrays.toString(offsetRecordByteArray),
-          Utils.getReplicaId(topicName, partition));
+          Utils.getReplicaId(topicName, partition),
```

Comment:
For not-hot-path logging it is always good to put exception as well, just curious did you find this issue during real debugging experience or you just happen to see it and change it?

Reply:
While debugging some oncall issues, there was no visibility as to why some method call fails. If being verbose you think might be an issue we can rate limit it.